conf/hb176rs_v4.conf

```diff
@@ -20,26 +20,19 @@
 ### Hardware checks
 ###
  * || check_hw_cpuinfo 2 176 176
+ * || check_hw_physmem 726513480kB 726513480kB 3%
  * || check_hw_swap 0kB 0kB 3%
  * || check_hw_ib 400 mlx5_ib0:1
  * || check_hw_eth lo
  * || check_hw_eth eth0
  * || check_hw_eth ib0
  * || check_hw_eth docker0
 
-
-#######################################################################
-###
-### Process checks
-###
- * || check_ps_loadavg 352
- * || check_ps_service -S -u root auditd
-
 #######################################################################
 ###
 ### Performance check 
 ###
- * || check_cpu_stream 580000
+ * || check_cpu_stream 650000
```

Comment:
Should this be more like 695000?

Reply:
The lowest value I saw was 678661.4. This was on your HX VM. I lowest on mine was 695304.1. The question would be when do we call it broken?

Reply:
Ok, sure that makes sense. Probably 650000 is enough to make it unlikely there is a problem.

Reply:
Addressed and new value discussed and changed